### PR TITLE
[GLUTEN-2699] Add some tests for DPP with scan disabled

### PR DESCRIPTION
**Pull request by PHILO-HE**
## What changes were proposed in this pull request?

This is just a follow up PR to add tests for https://github.com/oap-project/gluten/pull/2701.

### CODE REVIEW

File: gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala [hunk 1]
```diff
@@ -134,8 +134,10 @@ class VeloxTestSettings extends BackendTestSettings {
 
   enableSuite[GlutenDynamicPartitionPruningV1SuiteAEOff]
   enableSuite[GlutenDynamicPartitionPruningV1SuiteAEOn]
+  enableSuite[GlutenDynamicPartitionPruningV1SuiteAEOnDisableScan]
```
Comment (zhouyuan) [inline gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala:137]: so this is a new test suite, should also add AEOff?

**Comment by PHILO-HE** [comment]
@kerwin-zk, could you take a review?

File: gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala (hunk 1 above)
Comment (PHILO-HE) [reply gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala:137]: Yes, it is also worthy to add AQE off tests. Just added. Thanks!

**Comment by kerwin-zk** [comment]
> @kerwin-zk, could you take a review?

LGTM

**Review by zhouyuan — APPROVED** [review]

**Pull request merged**